src/ReinforcementLearningExperiments/src/experiments/MARL/IDQN_TicTacToe.jl

```diff
@@ -0,0 +1,96 @@
+# ---
+# title: JuliaRL\_IDQN\_TicTacToe
+# cover:
+# description: IDQN applied to TicTacToe competitive
+# date: 2023-07-03
+# author: "[Panajiotis Keßler](mailto:panajiotis.kessler@gmail.com)"
+# ---
+
+using StableRNGs
+using ReinforcementLearning
+using ReinforcementLearningBase
+using ReinforcementLearningZoo
+using ReinforcementLearningCore
+using Plots
+using Flux
+using Flux.Losses: huber_loss
+using Flux: glorot_uniform
+
+using ProgressMeter
+
+
+rng = StableRNG(1234)
+
+cap = 100
+
+RLCore.forward(L::DQNLearner, state::A) where {A <: Real} = RLCore.forward(L, [state])
```

Comment:
**NOTE:** This should be included into RLCore. For environments being identified by Reals, there should be a forward function for Approximators wrapping them into vectors s.t. they can be processed by neural networks.